DDG4/CMakeLists.txt

```
@@ -113,10 +113,31 @@ IF(TARGET EDM4HEP::edm4hep)
 ENDIF()
 
 IF(DD4HEP_USE_HEPMC3)
+  set(DDG4HepMC3_INCLUDES $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/hepmc> $<BUILD_INTERFACE:${HEPMC3_INCLUDE_DIR}>)
+  set(DDG4HepMC3_USES DD4hep::DDG4 Geant4::Interface ${HEPMC3_LIBRARIES})
+  set(DDG4HepMC3_DEFINITIONS)
+
+  IF(ZLIB_FOUND)
+    list(APPEND DDG4HepMC3_INCLUDES ${ZLIB_INCLUDE_DIR})
+    list(APPEND DDG4HepMC3_USES ${ZLIB_LIBRARIES})
```

```suggestion
    list(APPEND DDG4HepMC3_USES ZLIB::ZLIB)
```
Should be no need for the previous INCLUDE_DIRS